ui/templates/main.html

```diff
@@ -58,7 +58,8 @@ <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
     Load using:<br />
     http: <input type="radio" name="runner" value="http" checked/>
     (https insecure:<input type="checkbox" name="https-insecure" />,
-    standard go client instead of fastclient:<input type="checkbox" name="stdclient"/>)
+    standard go client instead of fastclient:<input type="checkbox" name="stdclient"/>,
+    resolvedIP: <input type="text" name="resolvedIP" size="12" value="" />)
```

Comment:
call it resolve, I try so far to have the url param name and the flag name be the same so it’s self documented